documentation/src/main/markdown/integrations/jenkinsplugin/jenkinsrequirements.md

```diff
@@ -1,6 +1,6 @@
 # Requirements for [detect_product_short] for Jenkins
 
-Requirements for [company_name] [detect_product_short] for Jenkins which may be in addition to those for [detect_product_short].
+Requirements for [detect_product_short] for Jenkins which may be in addition to those for [detect_product_short].
 
 * Access to the internet is required to download components from GitHub and other locations.
 * Minimum 8GB RAM.
```

Comment:
Can you change the minimum version for Jenkins required here to 2.426.3?

Reply:
Updated.